### PR TITLE
chore: add dbz-ui

### PR DESCRIPTION
**Pull request by indraraj**
We will deploy Debezium UI, and we will need the creation of a corresponding repo.
Debezium UI will be included as a federated module inside the COS UI. https://github.com/RedHatInsights/cloud-services-config/pull/612 will be the host app for this module.

can you assign this repo to @mdrillin @uidoyen and me

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -754,6 +754,10 @@ application-services:
       - id: cos
         title: "Connectors"
         group: overview
+        sub_apps:
```
Comment (Hyperkid123) [inline main.yml]: This file is no longer used for navigation and has no effect on it. Can you add an entry to `application-services-navigation.json`?

**Comment by Hyperkid123** [comment]
@indraraj can you send us the repository with the source UI code?

**Comment by indraraj** [comment]
> @indraraj can you send us the repository with the source UI code?

https://github.com/debezium/debezium-ui

File: main.yml (hunk 1 above)
Comment (indraraj) [reply main.yml]: Got it. Actually, this module is conditionally loaded from inside the COS UI.

Comment (Hyperkid123) [reply main.yml]: So it does not have an entry in the left nav?

Comment (indraraj) [reply main.yml]: yes

**Comment by Hyperkid123** [comment]
@indraraj can you give admin rights to me and @karelhala to the https://github.com/debezium/debezium-ui repository? We need it to properly set up the build repo.

**Comment by indraraj** [comment]
> @indraraj can you give admin rights to me and @karelhala to the https://github.com/debezium/debezium-ui repository? We need it to properly set up the build repo.

I will ask @gunnarmorling to give the access

**Comment by karelhala** [comment]
@indraraj if you want to have the proper experience with app deployment you can either add me to the repo or ping me and we can figure out travis together. That being said, the only thing we really need from you is actually the build repository, so if you want I can create the https://github.com/RedHatInsights/dbz-ui-build add you  @mdrillin @uidoyen to it and you can populate the build code there.

**Comment by indraraj** [comment]
> @indraraj if you want to have the proper experience with app deployment you can either add me to the repo or ping me and we can figure out travis together. That being said, the only thing we really need from you is actually the build repository, so if you want I can create the https://github.com/RedHatInsights/dbz-ui-build add you @mdrillin @uidoyen to it and you can populate the build code there.

Yes, I will add both of you to the repo. Thanks for creating https://github.com/RedHatInsights/dbz-ui-build

**Comment by uidoyen** [comment]
@karelhala I have pinged you on gchat.

**Pull request merged**